### DISCUSSION TITLE
I would like to translate the Buried Secrets expansion to polish

### DISCUSSION THREAD

**Discussion by Kesimux** [on3iro/aeons-end-randomizer#473] [Ideas]
Hi, 

I would like to translate the Buried Secrets expansion to polish, however I have no idea how to physically do it. 

Guidance would be appreciated :D

**Comment by on3iro** [comment]
Hi @Kesimux ,

sure thing - thanks for reaching out and wanting to contribute :)

First of all - do you have any experience in using a version control system like  [git](https://git-scm.com/)?
This would make it a bit easier to actually contribute.
Apart from using git itself you will need a text editor or IDE (e.g. Vim/Neovim, VSCode, IntelliJ, Notepad++, Sublime Text...).

Then you can basically follow this README:
https://github.com/on3iro/aeons-end-randomizer/tree/main/src/aer-data

Just skip the first part up to the headline **Add a new language**. (the readme is a bit outdated there)
Instead you clone the aeons-end-randomizer repository itself (the one where the [aer-data directory](https://github.com/on3iro/aeons-end-randomizer/tree/main/src/aer-data) is in).

Let me know if this information is enough for you to start working on the translation, or if you need more guidance :)

Thanks again for contributing :+1:

**Reply by Kesimux** [reply]
@on3iro I've translated the Buried Secrets expansion. Both player and nemesis cards.

Would it be possible for you to upload the files?

Thanks :D

[Buried Secrets - Pogrzebane Sekrety PL.zip](https://github.com/on3iro/aeons-end-randomizer/files/10609864/Buried.Secrets.-.Pogrzebane.Sekrety.PL.zip)

**Reply by on3iro** [reply]
@Kesimux awesome - thanks! I will make myself a reminder to push these changes asap. I am a bit busy right now, but I expect to have time for this in a couple of days. I'll let you know, when the release is ready :+1:

**Reply by Kesimux** [reply]
@on3iro No worries, thanks for the useful randomizer. I will also translate The New Age & Into The Wild once they arrive at my house 👍

**Reply by on3iro** [reply]
@Kesimux apparently another contributor also did the same work as you (but in a way, that's ready for me to merge)
https://github.com/on3iro/aeons-end-randomizer/pull/485

That's quite unfortunate - if you don't mind I will give credit to both of you inside the release notes and I will think of a way to communicate to contributors, that they should not start working on something without communicating it first. (if I remember correctly this is already part of our contribution guidelines, but maybe I should put this somewhere more prominent...)

**Reply by Gusti23** [reply]
@on3iro I didn't know that someone else is implementing polish translation into app. Sorry for duplicate. Also in this week I can create pull request with the translation of The New Age & Into The Wild if you want me to.

**Reply by Kesimux** [reply]
@on3iro No worries, :D I can translate The New Age & Into The Wild if you want to. Unless @Gusti23 wants to work on it instead, then that's fine too.

**Reply by on3iro** [reply]
Thanks folks, I just created a new release, which should be live in the next couple of minutes.

I don't really mind who of you two does the translations for New Age/Into The Wild - I am grateful either way :), so first come first server I guess ;)

**Reply by Kesimux** [reply]
@on3iro Hi, I think the patch didn't go through? Or perhaps the translation wasn't included :O

**Reply by on3iro** [reply]
What version does your randomizer show inside the drawer menu @Kesimux ?

**Reply by on3iro** [reply]
hm, wait I think you are right, something is wrong...

**Reply by Kesimux** [reply]
For me it's 3.16.00

**Reply by on3iro** [reply]
oooooh I see, there is an import missing.

@Gusti23 could you please make another pull request and add the import to `src/aer-data/src/PL/index.ts`

**Reply by Gusti23** [reply]
@on3iro I've prepared pull request with the fix for the missing import #489

**Reply by Gusti23** [reply]
@on3iro I've created pull request for translation of The New Age and Into The Wild. #492

**Reply by on3iro** [reply]
Awesome - thanks @Gusti23
